site/en/docs/web-platform/screen-sharing-controls/index.md

```diff
@@ -6,14 +6,14 @@ description: >
 authors:
   - beaufortfrancois
   - eladalon
-date: 2022-09-29
+date: 2022-09-30
 hero: image/vvhSqZboQoZZN9wBvoXq72wzGAf1/5VpvNzrvEM3qSxasqP1j.jpeg
 alt: toddler holding her lips photo
 tags:
   - chrome-107
 ---
 
-The [Screen Capture API] introduces additions to the existing Media Capture and Streams API to let the user select a screen or portion of a screen (such as a window) to capture as a media stream. This stream can then be recorded or shared with others over the network. This documentation introduces some changes to the API to better preserve privacy, and prevent accidental sharing of personal information.
+Sharing tabs, windows, and screens is already possible on the web platform with the [Screen Capture API]. In short, [`getDisplayMedia()`] allows the user to select a screen or portion of a screen (such as a window) to capture as a media stream. This stream can then be recorded or shared with others over the network. This documentation introduces some changes to the API to better preserve privacy, and prevent accidental sharing of personal information.
```

Comment:
I think that improves things significantly. Another problem in the same paragraph is "This documentation introduces..." It sounds a bit off. Like we're talking about that other spec (the document/documentation distinction is nuanced). Or like the current article is the normative reference.

Maybe:
"This **article** introduces some **recent** changes"
(The "recent" helps because it indicates this has already happened at a different time and place, and is discussed by the article.)

Reply:
Fixed.